angular/core/components/invitation_form/invitation_form.haml

```diff
@@ -18,6 +18,7 @@
 
     %label{for: "email-addresses", translate: "invitation_form.email_addresses"}
     %textarea.invitation-form__email-addresses.form-control#email-addresses{ng-model: "form.emails", ng-required: true, msd-elastic: "true", rows: 1, ng-model-options: "{debouce: 600}", ng-change: "storeDraft()", placeholder: "{{ 'invitation_form.email_addresses_placeholder' | translate }}" }
+    %validation_errors{subject: "form", field: "email"}
```

Comment:
Let's match this to the field name on the form, which is 'emails' (same as the ng-model above)

(We'll need to change the 422 response to reflect this as well)